apollo-integration/src/test/java/com/apollographql/apollo/ApolloWatcherTest.java

```diff
@@ -34,13 +33,11 @@
 public class ApolloWatcherTest {
   private ApolloClient apolloClient;
   private MockWebServer server;
-  private NormalizedCache normalizedCache;
-  private static final int TIME_OUT_SECONDS = 3;
+  private static final int TIME_OUT_SECONDS = 5;
```

Comment:
Fair enough -- creating a `MockWebServer` is an expensive operation. Wasn't sure how fast our CI instance was, but 2 seconds probably isn't a factor. I'll revert.